TwitchLib.PubSub/Events/OnChannelSubscriptionArgs.cs

```diff
@@ -4,6 +4,6 @@ namespace TwitchLib.PubSub.Events
 {
     public class OnChannelSubscriptionArgs
     {
-        public ChannelSubscription Subscription;
+        public ChannelSubscriptionEvent SubscriptionEvent;
```

Comment:
ReSharper sadly automatically changed the name Subscription to SubscriptionEvent. Meant to only rename the class.
Not sure if the Event Prefix is a good name anyhow, but do you have a better suggestion to make it clear that they're all of the same kind? Couldn't think of something better than Event at that specific moment, since it was like an 'event happening which Twitch notified us of'. And I just wanted all classes to have a consistent naming